src/DropDownMenu/DropDownMenu.js

```diff
@@ -344,6 +350,8 @@ class DropDownMenu extends Component {
       menuStyle = menuStyleProp;
     }
 
+    const iconButton = React.cloneElement(iconButtonElement);
```

Comment:
Well, because i did it in same manner, as in IconMenu https://github.com/callemall/material-ui/blob/master/src/IconMenu/IconMenu.js#L292